Add Vocalis and zathura, Handbrake is now GTK4 (#278)
myfreeweb/awesome-gtk: README.md

- [Simple Wireplumber GUI](https://github.com/dyegoaurelio/simple-wireplumber-gui) - WirePlumber (PipeWire session manager) GUI to rename devices and show properties. `#python` `#gtk4` `#libadwaita`.
- [SoundConverter](https://soundconverter.org) - Multithreaded sound converter with compatibility with everything that GStreamer reads and automatic renaming `#python` `#gtk3`.
- [Tagger](https://github.com/NickvisionApps/Tagger) - Music tag (metadata) editor `#csharp` `#gtk4` `#libadwaita`.

> - [Vocalis](https://gitlab.gnome.org/World/vocalis) - Sound recorder for the GNOME desktop `#gjs` `#gtk4` `#libadwaita`.

Reviewer: ```suggestion
- [Vocalis](https://flathub.org/apps/org.gnome.SoundRecorder) - Sound recorder for the GNOME desktop `#gjs` `#gtk4` `#libadwaita`.
```

Unless the GitLab has more information, I think it's better to default to the FlatHub link.

### Document Viewers

- [Evince](https://apps.gnome.org/Evince) - Document viewer for the GNOME desktop with support for PDF, Postscript, DjVu, comics etc. and SyncTex support with gedit `#c` `#gtk3` `#libhandy` `#gnome`.
- [Papers](https://apps.gnome.org/en/Papers) - Document viewer for the GNOME desktop (GTK 4 fork of Evince) with support for PDF, Postscript, DjVu, EPS, XPS and comics archives `#c` `#gtk4` `#libadwaita` `#gnome`.

> - [Xreader](https://github.com/linuxmint/xreader) - Generic document viewer with support for PDF, Postscript, djvu, comics and more `#c` `#gtk3` `#libxapp`.

Reviewer: ```suggestion
- [Xreader](https://github.com/linuxmint/xreader) - Generic document viewer with support for PDF, Postscript, DjVu, comics and more `#c` `#gtk3` `#libxapp`.
```

> - [zathura](https://pwmt.org/projects/zathura) - Customizable and functional document viewer with support for PDF, PostScript, DjVu and comic book `#c` `#gtk3`.

Reviewer: ```suggestion
- [zathura](https://pwmt.org/projects/zathura) - Minimalist and customizable document viewer with support for PDF, PostScript, DjVu and comic book `#c` `#gtk3`.
```

I would described it as "minimalist" rather than "functional".
